readme.md

```diff
@@ -214,14 +234,51 @@ There's no right way to do user stories. Their purpose is:
 
```

Comment:
I don't know if I agree with the above statement. While there certainly are many possible correct user stories for a given feature, there are also certainly very *wrong* ways to do user stories, which will fail to communicate your ideas, and fail to help you scope and understand your development efforts.